### ISSUE TITLE
Feature Request - SHOW_ALL_FILL scaleMode

### ISSUE THREAD

**Issue by hdodov** [photonstorm/phaser#2744]
A hybrid between the SHOW_ALL and RESIZE scale modes. Let's say I want a nice repetitive pattern as a background and my game centered in the middle.

If I use SHOW_ALL, the game is properly scaled on mobile, and works like a charm, but there is an empty and ugly square since the game is smaller than the window and the pattern isn't entirely over it.

If I use RESIZE, the pattern is all over the window, but the game isn't rescaled so that it fits inside and a big part of it isn't visible.

Take a look at this:

```
game.scale.scaleMode = Phaser.ScaleManager.RESIZE;
game.scale.onSizeChange.add(sizeChange);

function sizeChange(){
    var xRat = window.innerWidth/400, yRat = window.innerHeight/500;
    game.world.scale.setTo(Math.min(1, Math.min(xRat, yRat)));
}
```

My minimum game width and height are 400 and 500. I get the proportion between those values and the actual window values. Then, I set the game world's scale to the lesser of the two, or 1, if they are both greater than 1. This doesn't exactly work because the game world can't exactly be rescaled like this. In the console, when I open the game, it has the `scale` property, but not in the Phaser docs. If I use `game.world.scale.setTo()`, the game is scaled the way I want, but it starts flickering. Don't know if that's a bug or not.

**TL;DR** What I'm suggesting is a scale mode that rescales the game world so that it's visible inside the parent container, but in the same time sets the game size to that of the container. Or maybe a game world that can be rescaled properly by hand so that something like my code can work.

**Comment by photonstorm** [comment]
Thank you for taking the time to open this issue. However, official support for Phaser 2 has now ended as we focus on preparing Phaser 3 for beta release. In a bid to keep the project tidy, and avoid confusion, we are closing all v2 related issues.

This does not mean your issue won't be looked-at. In November 2016 the [Phaser CE (Community Edition)](https://github.com/photonstorm/phaser-ce) project began. The aim of Phaser CE was to allow the community to continue v2 development. Since then it has gained significant traction, with many updates and releases.

If you are still interested in having your issue investigated we strongly recommend testing the latest Phaser CE release. And only if the problem persists, opening an issue on the Phaser CE repo.
